pkg/controller/render/render_controller.go

```diff
@@ -457,6 +457,10 @@ func (ctrl *Controller) syncMachineConfigPool(key string) error {
 		return ctrl.syncFailingStatus(pool, fmt.Errorf("no MachineConfigs found matching selector %v", selector))
 	}
 
+	if err := AreGeneratedMCsUpdated(mcs); err != nil {
```

Comment:
Hmm, isn't this already being done via https://github.com/openshift/machine-config-operator/blob/main/pkg/controller/render/render_controller.go#L600-L607 in the `syncGeneratedMachineConfig` func below?